R/estSurfaces.R

```diff
@@ -19,14 +19,16 @@
 #' @param minNumObs numeric specifying the miniumum number of observations required to run the weighted regression, default is 100
 #' @param minNumUncen numeric specifying the minimum number of uncensored observations to run the weighted regression, default is 50
 #' @param edgeAdjust logical specifying whether to use the modified method for calculating the windows at the edge of the record. Default is TRUE.
+#' @param verbose logical specifying whether or not to display progress message
+#' @param interactive logical deprecated. Use 'verbose' instead
 #' @keywords water-quality statistics
 #' @return surfaces array containing the three surfaces estimated, array is 3 dimensional
 #' @export
 #' @examples
 #' eList <- Choptank_eList
 #' \dontrun{surfaces <- estSurfaces(eList)}
 estSurfaces<-function(eList, windowY=7,windowQ=2,windowS=0.5,
-                      minNumObs=100,minNumUncen=50,edgeAdjust=TRUE){
+                      minNumObs=100,minNumUncen=50,edgeAdjust=TRUE,verbose = TRUE, interactive=NULL){
```

Comment:
yeah, I went back and forth on that... my thought was since it was a "port" to the same functionality of a deprecated argument, it wouldn't hurt to add.